api/camunda/camunda-openapi.yaml

```diff
@@ -983,6 +1355,123 @@ components:
           format: int64
         tenantId:
           type: string
+    IncidentSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/IncidentFilterRequest"
+    IncidentFilterRequest:
+      type: object
+      properties:
+        key:
+          type: integer
+          format: int64
+        processDefinitionKey:
+          type: integer
+          format: int64
+        processInstanceKey:
+          type: integer
+          format: int64
+        type:
+          type: string
+        message:
+          type: string
+        flowNodeId:
+          type: string
+        flowNodeInstanceId:
+          type: string
+        state:
+          type: string
+        jobKey:
+          type: integer
+          format: int64
+        tenantId:
+          type: string
+        hasActiveOperation:
+          type: boolean
+    IncidentSearchQueryResponse:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryResponse"
+      type: object
+      properties:
+        items:
+          type: array
+          items:
+            $ref: "#/components/schemas/IncidentItem"
+    IncidentItem:
+      type: object
+      properties:
+        key:
+          type: integer
+          format: int64
+        processDefinitionKey:
+          type: integer
+          format: int64
+        processInstanceKey:
+          type: integer
+          format: int64
+        type:
+          type: string
+        message:
+          type: string
+        flowNodeId:
+          type: string
+        flowNodeInstanceId:
+          type: string
+        creationTime:
+          type: string
+        state:
+          type: string
+        jobKey:
+          type: integer
+          format: int64
+        tenantId:
+          type: string
+        hasActiveOperation:
+          type: boolean
```

Comment:
change-at-the-source, descriptions, new:

These properties all need descriptions. We should follow up with the original author of this endpoint to add them.

Reply:
Nudged in https://github.com/camunda/camunda-docs/pull/4217#discussion_r1746196922